shepherd_bms/src/analyzer.cpp

```diff
@@ -384,4 +384,18 @@ void Analyzer::disableTherms()
             }
         }
     }
+}
+
+void Analyzer::calcPackResistance(node *bms_data)
+{
+    if(bms_data->data.packCurrent >= 100)
+    {
+        for(int i = 0; i < NUM_CHIPS; i++)
+        {
+            for(int j = 0; j < NUM_CELLS_PER_CHIP; j++)
+            {
+                bms_data->data.chipData[i].cellResistance[j] =  (bms_data->data.chipData[i].openCellVoltage[j] - bms_data->data.chipData[i].voltageReading[j]) / bms_data->data.packCurrent;
```

Comment:
I'm unsure if it is wise to allow this to update instantly. This may have unintended consequences in terms of creating a loop where the change in resistance causes a change in allowed current and the change in current then causes a fluctuation in resistance and so on.
Safest bet would be to cache the calculated values and then implement them once the current drops back down to under 5 amps or under whatever threshold we consider is "motor off"